dangerfile.js

```diff
@@ -0,0 +1,71 @@
+import {message, danger, markdown} from "danger"
+
+const fs = require('fs').promises;
+const gzipSize = require('gzip-size');
+const git = require('simple-git/promise')(__dirname);
+
+// https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript
+function formatBytes(bytes, decimals = 2) {
+	if (bytes === 0) return '0 Bytes';
+
+	const k = 1024;
+	const dm = decimals < 0 ? 0 : decimals;
+	const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+	const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+}
+
+const percDiff =  (first, second) =>
+	`${(100 * Math.abs( (first - second) / ( (first+second)/2 ) )).toFixed(2)}%`
+
+const run = async () => {
+	const minified = danger.git.modified_files.filter(file => file.includes('.min.js'))
+
+	if (minified.length === 0) {
+		return markdown(`# No JS Changes`);
+	}
+
+	// Check if master exists & check it out if not.
+	const result = await git.branch(['--list', 'master']);
+	if (result.all.length === 0) {
+		await git.branch(['master', 'origin/master']);
```

Comment:
Oh good question, will need to test that.

Reply:
I made a little test PR (#2610) and `git remote -v` returned
```
origin	https://github.com/PrismJS/prism (fetch)
origin	https://github.com/PrismJS/prism (push)
```

So this will work even if the PR source is from a fork.